_targets/build.project.armv7m7-imxrt106x

```diff
@@ -40,24 +40,76 @@ KERNEL_OFFS=$((0x11000))
 # Project specific build
 #
 
-PROGS=("dummyfs" "imxrt-multi" "psh" "imxrt-flash")
-
-
-b_mkscript() {
+PROGS=()
+BOOT_DEVICE="flash1"
+NAME_USER_SCRIPT="user.plo"
+OFFS_USER_SCRIPT=$((0x10a00))
+MAGIC_USER_SCRIPT=
+
+# Pre-init script is launched before user script
+PREINIT_SCRIPT=("map itcm 0x0 0x40000 rwx"
+                "map dtcm 0x20000000 0x20028000 rw"
+                "map ocram2 0x20200000 0x2027fe00 rwx"
+                "map xip1 0x70000000 0x70400000 rx"
+                "map xip2 0x60000000 0x64000000 rx"
+                "phfs usb0 1.2 phoenixd"
+                "phfs flash0 2.0 raw"
+                "phfs flash1 2.1 raw"
+                "console 0.1"
+                "wait 2000")
+
+# Obligatory applications to run Phoenix-RTOS
+USER_SCRIPT=("kernel ${BOOT_DEVICE}"
+             "app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
+             "app ${BOOT_DEVICE} -x imxrt-multi ocram2 ocram2"
+             "app ${BOOT_DEVICE} -x psh xip1 ocram2"
+             "app ${BOOT_DEVICE} imxrt-flash ocram2 ocram2")
+
+
+b_mkscript_user() {
 	ksz=$((($(wc -c < "${PREFIX_PROG_STRIPPED}phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf") + SIZE_PAGE - 1) & PAGE_MASK))
 	poffs=$((KERNEL_OFFS + ksz))
 
-	for prog in "${PROGS[@]}"; do
-		psz=$((($(wc -c < "${PREFIX_PROG_STRIPPED}$prog") + SIZE_PAGE - 1) & PAGE_MASK))
-		printf "@%s(%x:%x)\n" "$prog" "$poffs" "$psz" >> "$PREFIX_BUILD/plo/script.plo"
-		((poffs+=psz))
+	printf "%08x" "$MAGIC_USER_SCRIPT" >> "$PREFIX_BUILD/plo/$NAME_USER_SCRIPT"
```

Comment:
can we have `\n` here - now  the file looks broken
![image](https://user-images.githubusercontent.com/3939983/119355508-a6dc1900-bca5-11eb-8ed5-b855c6ce0e1b.png)

Reply:
Yeah, it can be changed. 

However I would like to do it in separate PR because this change will affect plo which is currently in a merging process.

Reply:
I hoped that empty lines are allowed in plo script, so adding `\n` here would actually make first line of the script empty and be transparent for the parser

Reply:
Yes, you have right. The new line has been added after `MAGIC NUMBER`.